NEWS.md

```diff
@@ -0,0 +1,14 @@
+# rLakeAnalyzer 1.11.0
```

Comment:
:+1:

Reply:
Yeah I was unsure what to do with this and meant to flag it for discussion. The documentation is for several functions of which not a parameters are present in each function. Does this mean that each function has to have it's own documentation? Part of the issue is that parameters are passed `...` but that isn't recognized by roxygen. Any thoughts on an approach here? Is there a great desire to keep all these functions documented in one place?

Reply:
You could completely document the parameters across both functions, and then use `@aliases` so that both functions refer to the same documentation file. This is an easy way to not overly-repeat yourself. 

It looks like you've done that, so the next best thing would be to split out the documentation for those ts.* functions so you can specifically reference the underlying function in the `...` param info.

Reply:
Hmm, now I see the Roxygen issue. Falls into the usage section issue where Roxygen just mimics the documentation across all functions where alias is used. 

It looks like you can fix those build warnings @jread-usgs mentioned by using `@usage` tags in the documentation. Have you tried that?

Reply:
That sounds good to me.